### ISSUE TITLE
checkStyle is not enabled in CI

### ISSUE THREAD

**Issue by osana** [mapbox/mapbox-java#757]
CI Java code style check has the following output:
```:services-core:checkstyle NO-SOURCE
:services-directions:checkstyle NO-SOURCE
:services-geocoding:checkstyle NO-SOURCE
:services-geojson:checkstyle NO-SOURCE
:services-matching:checkstyle NO-SOURCE
:services-matrix:checkstyle NO-SOURCE
:services-optimization:checkstyle NO-SOURCE
:services-staticmap:checkstyle NO-SOURCE
:services-turf:checkstyle NO-SOURCE
```

On command line
```./gradlew checkstyle```  
and 
 ```make checkstyle``` 
(which is the same thing)  result in a successful pass because no source is selected to be checked.

CI should be running ./gradlew checkstyleMain  if we want to check source code

**Comment by osana** [comment]
interestingly `./gradlew build` does run the checkstyle somehow.. it runs it both on test and main
I was not ready to fix comments in test

**Comment by osana** [comment]
addressed by #756 
at least checkStyle is run on the src code,
checkstyle is not run on the tests though
